src/style.css

```
@@ -1 +1,88 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc) format('truetype');
+  font-weight: normal;
+}
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 20px 0;
+}
+
+.search__container {
+  width: 100%;
+  box-sizing: border-box;
+  padding-bottom: 20px;
+}
+
+.search__container:last-child {
+  padding-bottom: 0;
+}
+
+.large-search {
+  box-sizing: border-box;
+  width: 100%;
+  color: #3D4E61;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

move `font-family` into `html` selector
```suggestion
  font-family: inherit;
```